store/cassandra/store_cassandra.go

```diff
@@ -190,11 +190,15 @@ func NewCassandraStore(addrs, keyspace, consistency, CaPath, Username, Password,
 	}
 	cluster.Consistency = gocql.ParseConsistency(consistency)
 	cluster.Timeout = time.Duration(timeout) * time.Millisecond
+	cluster.ConnectTimeout = cluster.Timeout
```

Comment:
this seems useful, but warrants a separate commit so we can mark it in the changelog

Reply:
done